inst/report/scripts/generate_report.R

```diff
@@ -0,0 +1,705 @@
+# Author(s): Awa Synthia
+# Last modified: 2024
+
+# get fasta of pathogen and/or drug
+#' @export
+get_card_data <- function(pathogen = NULL, drug = NULL) {
+  destination_dir <- "CARD_data"
+  # Check if CARD data exists
+  if (!dir.exists(destination_dir)) {
+    dir.create(destination_dir)
+  }
+  if (!file.exists("CARD_data/aro_index.tsv")) {
+    # Step 1: Download CARD Data
+    download.file("https://card.mcmaster.ca/latest/data", "card_data.tar.bz2")
+    #unzip("card_data.tar.bz2", exdir = "CARD_data")
+    system(paste("tar -xjf card_data.tar.bz2 -C", destination_dir))
+  }
+
+  # Step 2: Open ARO_index.tsv
+  aro_data <- read.delim("CARD_data/aro_index.tsv", sep = "\t", header = TRUE)
```

Comment:
@AbhirupaGhosh @charmvang @epbrenner, if we were to reformat CARD ARO colnames per our styles/nomenclatures, what would this list look like? E.g., renaming Molecule -->  `drug` might make sense if that's what we use throughout. Please advise.